Libs/Widgets/ctkTransferFunctionItem.cpp

```diff
@@ -330,3 +332,57 @@ QVariant ctkTransferFunctionItem::itemChange(QGraphicsItem::GraphicsItemChange c
     }
   return res;
 }
+
+//-----------------------------------------------------------------------------
+void ctkTransferFunctionItem::mousePressEvent(QGraphicsSceneMouseEvent* event)
+{
+  QGraphicsObject::mousePressEvent(event);
+
+  ctkTransferFunction* tf = this->transferFunction();
+  if (tf)
+    {
+    ctkTransferFunctionRepresentation* tfRep = tf->representation();
+    Q_ASSERT(tfRep);
+    qreal x = tfRep->mapXFromScene(event->pos().x());
+    qreal y = tfRep->mapYFromScene(event->pos().y());
+    QPointF functionPosition = QPointF(x, y);
+    this->transferFunctionMousePressed(functionPosition, event->button());
```

Comment:
Shouldn't we use emit?

Reply:
Changed to use emit.

Does emit have any use besides as a style macro? The preprocessor directive for emit is empty.

Reply:
Yes, many people have noticed that emit is currently defined to empty string, but I haven't found in any official Qt documentation that it would be allowed to skip the emit keyword. They may change the internal implementation anytime to do something meaningful.

Also, it is important to show that it is not a method call (emitting a signal can cause multiple method calls, potentially asynchronously in a different thread, etc.).